lib/byron/1-percent.timelog

```diff
@@ -0,0 +1,4320 @@
+1589384183.268922344s -0
+1589384183.34260801s 1000
+1589384183.401558201s 2000
+1589384183.462808633s 3000
```

Comment:
Shouldn't be committed!

Reply:
That's for boosting GH statistics ;P

![Screenshot from 2020-05-14 16-53-04](https://user-images.githubusercontent.com/42900201/81949742-75952b80-9603-11ea-98e0-d2c53d98c3d2.png)

Reply:
I might have abused `git add lib` :smile: ...